### PR TITLE
5.x cleanup

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Controller/Controller.php [hunk 1]
```diff
@@ -191,14 +191,14 @@ public function __construct(
     ) {
         if ($name !== null) {
             $this->name = $name;
-        } elseif (!$this->name) {
+        } elseif (!isset($this->name)) {
```
Comment (othercorey) [inline src/Controller/Controller.php:194]: Static analyzers hate doing isset checks on non-nullable properties. Is this the route we want to go for typed properties?

Comment (ADmad) [reply src/Controller/Controller.php:194]: I had checked with phpstan's highest level and this change does not cause any error. There's nothing wrong in using `isset()` to check for initialization of non nullable properties IMO, especially when it's done in the constructor.

psalm does whine about this and report's it as redundant check. I had opened an issue for it but they said it's a design choice which won't be changed. I'll just disable the related rule for psalm once it's running again.

Comment (markstory) [reply src/Controller/Controller.php:194]: Isn't using isset() going to result in different behavior than before? Previously this block would run if `name = ''` which is the current default. Now this block will not run and $name won't get a fallback value based on the `controller` parameter.

Comment (ADmad) [reply src/Controller/Controller.php:194]: I have also removed the default empty string for the property.

Comment (markstory) [reply src/Controller/Controller.php:194]: Oops I missed that :cry:

**Review by markstory — APPROVED** [review]

**Pull request merged**